### PR TITLE
Add discord link

### LINKED ISSUE #100: Home画面にDiscordへのリンクを追加する
Figma:
https://www.figma.com/file/x71sECvdnsw8RTfKG0E4fB/FlutterKaigi-2023-App?type=design&node-id=54012-28023&mode=design&t=O7hayytZtUDgseYw-0

Link url:
https://discord.com/invite/Nr7H8JTJSF

### PR DESCRIPTION
**Pull request by nitakan**
## Overview
close #100 

Add discord link to home screen.

## Changes

- Added `discord` localization support to existing localization classes.
- Implemented an outlined Discord button on the home screen.
- Changed the display order of Medium and GitHub buttons on the home screen.
- Modified line spacing and alignment in the `Wrap` class.

### Web
image

### CODE REVIEW

File: lib/ui/screen/home.dart [hunk 1]
```diff
@@ -97,6 +99,13 @@ class HomePage extends ConsumerWidget {
                 },
                 child: Text(localization.twitter),
               ),
+              OutlinedButton(
+                onPressed: () async {
+                  final uri = Uri.parse('https://medium.com/flutterkaigi');
+                  await launchInExternalApp(uri);
+                },
+                child: Text(localization.medium),
+              ),
```
Comment (nitakan) [inline lib/ui/screen/home.dart:108]: Swapped the positions of the Twitter and GitHub buttons to align with the design.

Previous: 



Figma:

**Comment by koji-1009** [comment]
Hi @nitakan 
Thank you for your contribution!

The implementation looks good, but the formatting seems incomplete. Could you please format it using `dart format` or something similar?

**Comment by nitakan** [comment]
Hi @koji-1009 

Thank you for the feedback!
I've reformatted the code using `dart format` .
Please let me know if there are any other changes needed.

**Review by koji-1009 — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**